inc/patterns/footer-title-tagline-social.php

```diff
@@ -1,84 +1,26 @@
 <?php
```

Comment:
This diff looks deceptively complex. This pattern is being replaced with the one that was previously in the `footer-title-tagline-social-dark.php` file, so that's why it looks like so much has changed.